### ISSUE TITLE
include information about the currently used JDBC driver into system information section and somehow in the shell

### ISSUE THREAD

**Issue by aisbergde** [schemacrawler/SchemaCrawler#428]
# New Feature Request

It could help debugging to know which JDBC driver is currently used

**Describe the solution you'd like**

include information about the JDBC driver into system information section 

```
System Information
========================================================================

generated by                              SchemaCrawler 16.14.1
generated on                              2021-03-16 23:36:20.3299206

here in this section could be some information about the used JDBC driver
```

allow to get this information in any way also in the shell

**Describe alternatives you've considered**

I have no idea how to detect the currently used JDBC driver

**Comment by sualeh** [comment]
@aisbergde - strangely enough, this information used to be displayed. Please use `--log-level=CONFIG` on the command-line to see the driver. 

I am leaving this issue open, since I want add this information to the "sys" command in the shell.

**Comment by aisbergde** [comment]
The issue is, this is all documentation in the online help about the log level:

```
Shell Command:
log

Options:
      --log-level=<loglevel>
         Set log level using one of OFF, SEVERE, WARNING, INFO, CONFIG, FINE,
           FINER, FINEST, ALL
         Optional, defaults to OFF
```

you as the developer know, what is contained in different levels. A normal user doesn't know this.

Some of my feature requests are based on the "Experience of a new user who is not yet blind to operations". And I'm just trying with my ideas to make it easier for me and others to get started.

In all bug entries the JDBC version is queried. Wouldn't it be easiest to just output this right away, instead of hoping that a user guesses that he might use `--log-level=CONFIG`?

**Comment by sualeh** [comment]
In the next SchemaCrawler release, you will be able to get this information from the SchemaCrawler interactive shell with `sys -C`. 

If you want it otherwise, you will need to use the following options on the command-line `--info-level=maximum --command=details`. Otherwise, you can use a log level to find this information. In any case, information on the driver is well documented on the [database support page](https://www.schemacrawler.com/database-support.html).

**Comment by aisbergde** [comment]
I found that all drivers are in the SchemaCrawler lib directory. And it looks like you also know the content of this folder per version. In other words, if a user uses the SC download then you know which driver is used by default, if the user will write SC version and database in the bug report.

I created this feature request before I found the location of the lib directory. And I did not find it so quick, because I used chocolatey. And that's why I created #431 to make this easier to find for other users.

I think this `sys -C` should be fine, if it will be described in the interactive help file.
